mesonbuild/build.py

```diff
@@ -140,7 +141,7 @@ def __init__(self, environment):
         self.dep_manifest_name = None
         self.dep_manifest = {}
         self.cross_stdlibs = {}
-        self.test_setups = {}
+        self.test_setups = {}                         # type: typing.Dict[str, TestSetup]
```

Comment:
Big indent here on purpose?

Reply:
Yeah, I wanted to be able to line all of the type annotations for those attributes up like:

```python
self.foo = []                     # type: ...
self.really_long_name = []        # type: ...
```